client/signup/steps/intent/index.tsx

```diff
@@ -30,10 +36,15 @@ export default function IntentStep( props: Props ): React.ReactNode {
 	const branchSteps = useBranchSteps( stepName );
 
 	const submitIntent = ( intent: IntentFlag ) => {
-		branchSteps( EXCLUDE_STEPS[ intent ] );
-		recordTracksEvent( 'calypso_signup_intent_select', { intent } );
-		dispatch( submitSignupStep( { stepName }, { intent } ) );
-		goToNextStep();
+		recordTracksEvent( 'calypso_signup_select_intent', { intent } );
```

Comment:
Hmmm...I just found that the event name changed back to the one before renaming...